Formula/i/irssi.rb

```diff
@@ -33,11 +33,9 @@ def install
       --disable-dependency-tracking
       --prefix=#{prefix}
       --sysconfdir=#{etc}
-      --with-bot
       --with-proxy
       --enable-true-color
       --with-socks=no
-      --with-perl=no
```

Comment:
can you link the upstream changes for these? Thanks!

Reply:
also, maybe you need

```
      --with-perl=yes
      --with-perl-lib=#{lib}/perl5/site_perl
```


and

```
    # This is not how you'd use Perl with Irssi but it is enough to be
    # sure the Perl element didn't fail to compile, which is needed
    # because upstream treats Perl build failures as non-fatal.
    # To debug a Perl problem copy the following test at the end of the install
    # block to surface the relevant information from the build warnings.
    ENV["PERL5LIB"] = lib/"perl5/site_perl"
    system "perl", "-e", "use Irssi"
```

see #136747